src/Modules/OrchardCore.Commerce/Controllers/ShoppingCartController.cs

```diff
@@ -159,11 +159,11 @@ public async Task<ActionResult> Update(ShoppingCartUpdateModel cart, string shop
             // Preserve invalid lines in the cart, but modify their Quantity values to valid ones.
             if (!isValid)
             {
-                var minOrderQuantity = (await _productService.GetProductAsync(line.ProductSku))
-                    .As<InventoryPart>().MinimumOrderQuantity.Value;
+                var minOrderQuantity = (await _productService.GetProductAsync(line.ProductSku))?
+                    .As<InventoryPart>()?.MinimumOrderQuantity.Value;
```

Comment:
If you add a null-coalescing operator at the end, then this variable won't become nullable.

```suggestion
                var minOrderQuantity = (await _productService.GetProductAsync(line.ProductSku))?
                    .As<InventoryPart>()?.MinimumOrderQuantity.Value ?? 0;
```